doc/classes/InputEventKey.xml

```diff
@@ -79,7 +79,7 @@
 		</member>
 		<member name="physical_keycode" type="int" setter="set_physical_keycode" getter="get_physical_keycode" enum="Key" default="0">
 			Represents the physical location of a key on the 101/102-key US QWERTY keyboard, which corresponds to one of the [enum Key] constants.
-			To get a human-readable representation of the [InputEventKey], use [code]OS.get_keycode_string(event.keycode)[/code] where [code]event[/code] is the [InputEventKey].
+			To get a human-readable representation of the [InputEventKey], use [code]OS.get_keycode_string(DisplayServer.keyboard_get_keycode_from_physical(event.physical_keycode))[/code] where [code]event[/code] is the [InputEventKey].
```

Comment:
Probably should be just `[code]OS.get_keycode_string(event.physical_keycode)[/code]` since it's referring to the human-readable representation of a physical key, not a remapped key.

Reply:
Or this portion should be left only for the `keycode` and removed from `physical_keycode` and `key_label` at all.

Reply:
If you want to print what the key for the user is, though, that's what you need to do, which isn't shown in the InputEventKey documentation.  I imagine that's the common use case, though perhaps it should be worded differently.

For example, if you have an event on the physical location of the Q key (right under the 1), but the user has an AZERTY keyboard instead of QWERTY, you'd want to display "A" instead of "Q". Ex: "Press [Q] to use action" vs "Press [A] to use action."  If you're not using QWERTY, the keys will display incorrectly.

It definitely needs to be documented in some easily accessible way.

Reply:
When it's difficult to describe what the heck is going on, over something so practically simple, that warrants adding a new helper method. At least, that's what it looks like to me every time *this* happens.

Ideally and personally, for this PR, it'd be nice to directly to directly lead the user to the methods via links, instead of using the code snippet. That way, the note is less "Just do this, trust me" and more "Here's why you're doing this" because the related methods offer more information to whoever is interested. But, it's more important to be correct, rather than being awkward to read.

Reply:
Linking is not a bad idea.  Can that be done directly in the code snippit?  It might get too wordy otherwise.  I do feel like the names of the functions are somewhat self explanatory: keyboard_get_keycode_from_physical gets the keycode from the physical key and get_keycode_string gets the string from the keycode.  Only thing that's not explained, and I don't know why this is to explain it, is why the keyboard_get_keycode_from_physical is on the DisplayServer and not the OS or Input related classes. 🤔

Reply:
You can't link from a snippet. And being wordy is not a problem, better than being terse. This snipped should also be formatted properly as a code block, since it's really long.

```suggestion
			To get a human-readable representation of the [InputEventKey], use [method OS.get_keycode_string] in combination with [method DisplayServer.keyboard_get_keycode_from_physical]:
			[codeblocks]
			[gdscript]
			# event is InputEventKey.
			var keycode = DisplayServer.keyboard_get_keycode_from_physical(event.physical_keycode)
			OS.get_keycode_string(keycode)
			[/gdscript]
			[/codeblocks]
```

Plus the C# example.